bluepysnap/frame_report.py

```diff
@@ -111,6 +112,59 @@ def get(self, group=None, t_start=None, t_stop=None):
         return res
 
 
+class FilteredFrameReport(object):
+    """Access to filtered FrameReport data."""
+    def __init__(self, frame_report, group=None, t_start=None, t_stop=None):
+        """Initialize a FilteredFrameReport.
+
+        A FilteredFrameReport is a lazy and cached object which contains the filtered data
+        from all the populations of a report.
+
+        Args:
+            frame_report (FrameReport): The FrameReport to filter.
+            group (None/int/list/np.array/dict): Get spikes filtered by group. See NodePopulation.
+            t_start (float): Include only frames occurring after this time.
```

Comment:
Strictly speaking, the times t_start and t_stop would be excluded. Is it the case?

Reply:
no indeed this is >=. 
But this is tricky since we talk about floats here. 
I changed the docstrings accordingly